probe-rs/src/debug/debug_step.rs

```diff
@@ -262,7 +262,7 @@ impl SteppingMode {
                         if function.attribute(gimli::DW_AT_noreturn).is_some() {
                             return Err(DebugError::Other(anyhow::anyhow!(
                                 "Function {:?} is marked as `noreturn`. Cannot step out of this function.",
-                                function.function_name(debug_info)
+                                function.function_name(debug_info).unwrap_or("<unknown>".to_string())
```

Comment:
Please prefer `as_deref()` over `to_string()`

Reply:
Nice ... I won't go to bed stupid tonight :)